decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,10 +1,46 @@
 <div class="wrapper">
   <div class="row">
     <div class="large-8 large-centered">
-    <div class="static__message">
-      <h1 class="heading1 page-title"><%= t(".title") %></h1>
-      <p><%= t(".try_later") %></p>
+      <div class="static__message">
+        <h1 class="heading1 page-title"><%= t(".title") %></h1>
+        <p><%= t(".try_later") %></p>
+        <div class="table-scroll">
+          <table class="table-list">
+            <tr>
```

Comment:
Could you just iterate over the `@info_hash` and generate the table rows dynamically?

There is one issue still with the `<th>` elements that you are adding an unnecessary interpolation variable to the `t(...)` call. But if you do it the same way for the table here as you are already doing for the plain info, I think you should be able to get rid of that issue.